### ISSUE TITLE
Panels and Tabs on a Resource View overlap

### ISSUE THREAD

**Issue by mighani** [avo-hq/avo#1326]
I have this code in my resource file:

```
tabs do
    tab "Basis Info" do
      panel do
        field :salutation, as: :select, enum: ::Employee.salutations,
              name: "Anrede", display_with_value: true, placeholder: 'Anrede'
        field :complex_name, as: :text, hide_on: :all, as_label: true do |model|
          "#{model.first_name}  #{model.last_name}"
        end
      end
    end
    tab "Kontakt Info" do
      panel do
        field :email, as: :text, name: "Email"
        field :phone, as: :text, name: 'Telefon'
        field :gvcv_url, as: :text, name: 'GVCV Url'
      end
    end
    tab "Einstellung Info" do
      panel do
        field :start_date, as: :date, name: "Einstelldatum",sortable: true
        field :resign_date, as: :date, name: "Kündigungsdatum",sortable: true
        field :end_date, as: :date, name: "Letzter Arbeitstag",sortable: true
        field :active, as: :boolean, sortable: true,name: 'Aktiv'
      end
    end
    tab "Anhänge und Notizen" do
      panel do
        field :note, as: :trix, name: 'Notizen', attachments_disabled: true
        field :documents, as: :files, visible: -> (resource:) { context[:user].hr? || context[:user].admin?  }
        field :meta, as: :key_value, key_label: "Attribut", value_label: "Attribut Wert", action_text: "Neues Attribut", delete_text: "Attribut Löschen", disable_editing_keys: false, disable_adding_rows: false, disable_deleting_rows: false
      end
    end
  end

  panel do
    tabs do
      field :project, searchable: true, as: :has_many, name: 'Projekt'
      field :salaries, as: :has_many, name: "Gehalt", visible: -> (resource:) { context[:user].hr? || context[:user].admin?  }
    end
  end
```

On the rendered page the panel overlaps the next one, if the field list is longer than the panel height: 

![Image 2022-10-18 at 4 38 24 PM](https://user-images.githubusercontent.com/782174/196463332-2030d0fe-853f-4b45-8564-a279595f1947.jpg)

Am I missing something?

**Comment by adrianthedev** [comment]
What Avo version are you using?

**Comment by mighani** [comment]
sorry,  v2.16.0

**Comment by adrianthedev** [comment]
Can you try `2.17.0`? We pushed [this fix](https://github.com/avo-hq/avo/pull/1299).

**Comment by adrianthedev** [comment]
Closing for now. Feel free to reopen if it's still broken.

**Comment by mighani** [comment]
Thanks!! It was resolved in 2.17.0
